<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HashRenamer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.MainWindow {background-color: #1abc9c}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addFilesPushButton">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>155</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color: #2980b9;
	border-radius: 10px;
	color: #ffffff;
	font-size: 14px;
	font-weight: bold;
}
.QPushButton:hover {
	background-color: #3498db;
}</string>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDirectoriesPushButton">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>155</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color: #2980b9;
	border-radius: 10px;
	color: #ffffff;
	font-size: 14px;
	font-weight: bold;
}
.QPushButton:hover {
	background-color: #3498db;
}</string>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectedFileOrDirectoryLineEdit">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>85</y>
      <width>550</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLineEdit {
	background-color: rgba(255, 255, 255, 0);
	border: none;
	color: #ffffff;
	font-size: 14px;
	font-weight: bold;
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>85</y>
      <width>570</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QProgressBar::chunk {
	background-color: rgb(85, 170, 255);
	border-radius: 4px;
	margin: 2px;
}
.QProgressBar {
	color: #ffffff;
	border: none;
	background-color: #2c3e50;
}</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="runPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>155</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color: #2980b9;
	border-radius: 10px;
	color: #ffffff;
	font-size: 14px;
	font-weight: bold;
}
.QPushButton:hover {
	background-color: #3498db;
}</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>550</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 73, 94);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sha1RadioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>SHA-1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="md5RadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>155</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color: #2980b9;
	border-radius: 10px;
	color: #ffffff;
	font-size: 14px;
	font-weight: bold;
}
.QPushButton:hover {
	background-color: #3498db;
}</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>155</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color: #2980b9;
	border-radius: 10px;
	color: #ffffff;
	font-size: 14px;
	font-weight: bold;
}
.QPushButton:hover {
	background-color: #3498db;
}</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <zorder>progressBar</zorder>
   <zorder>selectedFileOrDirectoryLineEdit</zorder>
   <zorder>addFilesPushButton</zorder>
   <zorder>addDirectoriesPushButton</zorder>
   <zorder>runPushButton</zorder>
   <zorder>statusLabel</zorder>
   <zorder>sha1RadioButton</zorder>
   <zorder>md5RadioButton</zorder>
   <zorder>stopPushButton</zorder>
   <zorder>pausePushButton</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
